### PR TITLE
[IBCDPE-534] Add develop  s3 Ingestion bucket

### PR DESCRIPTION
**Pull request by thomasyu888**
Purpose: This creates the ingestion bucket for pilot data. (SYNAPSE_INPUT_SHARED_BUCKET)

This is the first bucket in the ingress pipeline for RECOVER project. This will be the bucket that the digital health team uses to deposit pilot data.

### CODE REVIEW

File: config/develop/s3-ingestion.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  # This is the shared account (credentials shared on Last Pass) for RECOVER
+  CostCenter: "MGH RECOVER / 122500"
+parameters:
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Allow accounts, groups, and users to access bucket (default is no access).
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+    - 'arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/meghasyam@sagebase.org'
+    - 'arn:aws:iam::621233246578:role/EES3-RK-6A32F45B-RECOVER' # QA ARN role for Care Evolution
```
Comment (thomasyu888) [inline config/develop/s3-ingestion.yaml:19]: We can remove access to the Care Evolution role.

File: config/develop/s3-ingestion.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.0/templates/S3/synapse-external-bucket.j2"
+stack_name: "dev-recover-ingestion"
+stack_tags:
+  # This is the shared account (credentials shared on Last Pass) for RECOVER
+  CostCenter: "MGH RECOVER / 122500"
+parameters:
+  # (Optional) Name of the created bucket.
+  BucketName: "dev-ingestion-prod"
```
Comment (thomasyu888) [inline config/develop/s3-ingestion.yaml:13]:
```suggestion
  BucketName: "recover-dev-ingestion"
```

File: config/develop/s3-ingestion.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# NOTE:
+# This is the first bucket in the ingress pipeline for RECOVER project.
+# Ingestion bucket or SYNAPSE_INPUT_SHARED_BUCKET
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.0/templates/S3/synapse-external-bucket.j2"
+stack_name: "dev-recover-ingestion"
```
Comment (thomasyu888) [inline config/develop/s3-ingestion.yaml:7]:
```suggestion
stack_name: "recover-dev-ingestion-bucket"
```

File: config/develop/s3-ingestion.yaml (hunk 1 above)
Comment (rxu17) [reply config/develop/s3-ingestion.yaml:20]: Based on what we discussed, should we just remove that now?

**Review by rxu17 — APPROVED** [review]
LGTM! Just a small comment

File: config/develop/s3-ingestion.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+parameters:
+  # (Optional) Name of the created bucket.
+  BucketName: "recover-dev-ingestion"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Allow accounts, groups, and users to access bucket (default is no access).
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+    - 'arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/meghasyam@sagebase.org'
+    - 'arn:aws:iam::621233246578:role/EES3-RK-6A32F45B-RECOVER' # QA ARN role for Care Evolution
```
Comment (thomasyu888) [inline config/develop/s3-ingestion.yaml:20]:
```suggestion
```

**Pull request merged**